### PR TITLE
Gk/title matches api name/w 7275845

### PR DESCRIPTION
**Pull request by gkhunger-sf**
- Adds a rule for title valdiation - title should match separated and capitalized api name
- Splits the base uri rule into two - existence and pattern match

### CODE REVIEW

File: test/mercury/title.tests.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*
```
[the remark this answers is not in the document]
Comment (gkhunger-sf) [reply test/mercury/title.tests.ts:1]: good catch!

**Comment by monkey-jeff** [comment]
DO NOT MERGE!

So, this doesn't quite match what was intended by the standard.  

I Have just discussed with the C4E team around this and while the 1.0 standard does specify that `the title MUST match the root-aggregate (read as API name)` in the baseUri, there have been examples where this is NOT the case, especially when talking about contractions and the current CDN APIs.

This constraint is being changed to a RECOMMENDED in the 1.1 standards and we cannot reliably lint for it without breaking some product decisions around naming.  (I believe the CDN API is an example of this already)

The splitting of the baseUri enforcement to make it more readable is still a good change, but I don't think we can enforce the matching of the API name in the baseUri as the true constraint here that is not changing it `The Title MUST match the Api Name in exchange` which we cannot lint for.

**Comment by gkhunger-sf** [comment]
That's a bummer. Well, I will close this PR and create a new one for just splitting the baseUri rule. Is there anything else that's coming up in 1.1 that impacts our other linter stories in the pipeline?

**Comment by monkey-jeff** [comment]
> That's a bummer. Well, I will close this PR and create a new one for just splitting the baseUri rule. Is there anything else that's coming up in 1.1 that impacts our other linter stories in the pipeline?

Not that I know of.  But will keep on top of it if that changes

**Pull request closed without merging**